dlt/destinations/sql_jobs.py

```diff
@@ -200,19 +205,63 @@ def gen_delete_temp_table_sql(
             sql.append(f"INSERT INTO {temp_table_name} SELECT {unique_column} {clause};")
         return sql, temp_table_name
 
+    @classmethod
+    def gen_select_from_dedup_sql(
+        cls,
+        table_name: str,
+        primary_keys: Sequence[str],
+        columns: Sequence[str],
+        sort_column: str = None,
+        condition: str = None,
+        condition_columns: Sequence[str] = None,
+    ) -> str:
+        """
+        Returns SELECT FROM statement where the FROM clause represents a deduplicated version of the `table_name` table.
+        Expects column names provided in arguments to be escaped identifiers.
```

Comment:
please explain what is the condition and condition columns and how they are related, you use this function in two ways:
 to get temp table and to gen full select. this is also worthy of mention.